java/src/jmri/jmrix/ncemonitor/NcePacketMonitorBundle.properties

```diff
@@ -1 +1,38 @@
-Title   = DCC Packet Analyzer
+Title              = DCC Packet Analyzer
+PacketAnalyzer     = Original Packet Analyzer
+DccMeter/Analyzer  = DccMeter/Analyzer (2023)
+
+UnitHelpButtonLabel       = Unit help
+UnitHelpButtonToolTip     = Shows the help information from the analyzer
+UnitHelpPageButtonLabel   = More help
+UnitHelpPageButtonToolTip = Pages the unit help screens
+DupFilterCheckBoxLabel   = Hide duplicates
+DupFilterCheckBoxToolTip = Shows duplicates with count
+PortBoxToolTip     = Select the port to use
+ModelBoxLabel      = Select model of analyzer:
+ModelBoxToolTip    = New model has LCD display and USB, Old model was just a PC board
+OpenButtonLabel    = Open
+OpenButtonToolTip  = Configure program to use selected port
+CloseButtonLabel   = Close
+CloseButtonToolTip = Release the selected port and stop monitoring
+VerboseButtonLabel = Verbose
+VerboseButtonToolTip = Shows repeated packet with count of repeats
+Hex0Label          = Hex with preamble symbol
+Hex1Label          = (as above with spaces)
+Hex2Label          = Hex without preamble symbol
+Hex3Label          = (as above with spaces)
+Hex4Label          = Hex with preamble count in hex
+Hex5Label          = (as above with spaces)
+AccOffLabel        = Hide acc packets
+AccOnLabel         = Show acc packets
+IdleOffLabel       = Hide idle packets
+IdleOnLabel        = Show idle packets
+LocoOffLabel       = Hide loco packets
+LocoOnLabel        = Show loco packets
+ResetOffLabel      = Hide reset packets
+ResetOnLabel       = Show reset packets
+SignalOffLabel     = Hide signal packets
+SignalOnLabel      = Show signal packets
+AccSingleLabel     = Acc addresses single
+AccPairedLabel     = Acc addresses paired
+SerialPortLabel    = Serial port: 
```

Comment:
There is no empty line at the end of this file. We want an empty line at the end of the file.